test/e2e/framework/pods.go

```diff
@@ -146,6 +149,27 @@ func (c *PodClient) Update(name string, updateFn func(pod *v1.Pod)) {
 	}))
 }
 
+// AddEphemeralContainerSync adds an EphemeralContainer to a pod and waits for it to be running.
+func (c *PodClient) AddEphemeralContainerSync(pod *v1.Pod, ec *v1.EphemeralContainer, timeout time.Duration) {
+	namespace := c.f.Namespace.Name
+
+	podJS, err := json.Marshal(pod)
+	ExpectNoError(err, "error creating JSON for pod: %v", err)
```

Comment:
`err` is already a first parameter in `ExpectNoError`, no need to include it into the error message. `pod` name might be more informative in the error description.

Reply:
Same for other occurrences

Reply:
Good point, thanks.